package.json

```diff
@@ -18,7 +18,7 @@
     "dev": "rollup --config --watch",
     "lint": "eslint src/ test/ --ignore-pattern test/browser/testcases.js",
     "fix-lint": "eslint src/ test/ --fix --ignore-pattern test/browser/testcases.js",
-    "test": "mocha out/test/*.test.{js,cjs,mjs} --parallel",
+    "test": "mocha out/*.test.{js,cjs,mjs} --parallel",
```

Comment:
previously we use `tsc` to compile both src and test folders into out folder, thus we have this "test/" here. Now why it can be omitted? I assume there should be some change on tsconfig, e.g. only to compile test code into out folder.

Reply:
The tsconfig remains the same. Previously, test code rely on src code, they will be compiled with the folder structure. Now, test code import the fm module from ../, it will use the entry point specified in package.json which is dist/commonjs/index.js So only the test code will be compiled. There will be no folder structure. It just complied any ts files in test folder and put the result in out folder.

Reply:
Got it. We only TS compile test code, in L16. Thanks for clarification.